stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/ChannelListDiffCallback.kt

```diff
@@ -47,7 +48,8 @@ internal class ChannelListDiffCallback @JvmOverloads constructor(
             name = !channelNameIsTheSame(newChannel, oldChannel),
             avatarView = !channelUsersAreTheSame(newChannel, oldChannel),
             readState = channelUserReadIsDifferent(oldChannel, newChannel),
-            lastMessageDate = !channelLastMessageDatesAreTheSame(oldChannel, newChannel)
+            lastMessageDate = !channelLastMessageDatesAreTheSame(oldChannel, newChannel),
+            unreadCount = unreadCountIsDifferent(oldChannel, newChannel)
```

Comment:
Finally, a suggestion you'll be able to apply via GitHub because it doesn't change imports 😄 

```suggestion
            unreadCount = unreadCountIsDifferent(oldChannel, newChannel),
```